src/storage/write_ahead_log/log_serializer_task.cpp

```diff
@@ -22,6 +22,7 @@ void LogSerializerTask::Process() {
   {
     common::SpinLatch::ScopedSpinLatch guard(&flush_queue_latch_);
     temp_flush_queue = std::move(flush_queue_);
+    flush_queue_ = std::queue<RecordBufferSegment *>();
```

Comment:
This is an actual fix riding on the rest of the PR, right?

Reply:
Yep. It complained about flush_queue_ being looked at again in the calling context, however a moved from container is left in a valid but otherwise non-deterministic state. Since std::queue doesn't have a clear or reset method, the solution is just a new object I guess.